src/screens/Library/components/InstallModal/index.tsx

```diff
@@ -545,7 +545,7 @@ export default function InstallModal({
                       <span className="warning">
                         {`${t(
                           'install.path-not-writtable',
-                          'Warning: path might not be writable.'
+                          `Warning: path doesn't exists or might not be writable.`
```

Comment:
We should show the actual error and not if the path might not exist or is not writeable. We had a error where a user did not had powershell on path and we didn't show the error of check-disk-space in the frontend!

Reply:
But how heroic even worked if he did not have Powershell?
We use it for everything legendary/gogdl related. Even login won't be possible.